airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2871,6 +2917,33 @@ components:
       properties:
         consentUrl:
           type: string
+    OAuthSourceRequestBody:
+      type: object
+      required:
+        - authCode
```

Comment:
this is _usually_ called `code` but maybe we just shouldn't specify it and rely on additionalProperties